integration-tests/coverage_py/Dockerfile

```diff
@@ -29,7 +29,8 @@ RUN git config --global user.name "Your Name"
 RUN git add ignore.me
 RUN git commit -m "testing"
 
+WORKDIR ..
 ENV CC_TEST_REPORTER_ID=c4881e09870b0fac1291c93339b36ffe36210a2645c1ad25e52d8fda3943fb4d
-RUN test-reporter format-coverage -d
+RUN test-reporter format-coverage -d -t coverage.py python-test-reporter/coverage.xml
```

Comment:
is there any value to having this as a separate test case (in addition to the previous one) vs. just modifying the existing one?

Reply:
@chrishulton this is more of like an end-to-end test. We don't run it on CI because it takes a long time to run all the `integration-tests` but i do find it helpful to run locally particularly when changing something language focused.